plugin/pxr/maya/lib/usdMaya/CMakeLists.txt

```diff
@@ -6,7 +6,6 @@ pxr_shared_library(${PXR_PACKAGE}
         gf
         js
         kind
-        plug
         sdf
         tf
         usd
```

Comment:
Of the libraries still listed here, it looks like you could now also remove these if you wanted to:

```
js
usdLux
usdRi
usdShade
usdSkel
```

Reply:
Done. Please see f70e798850e98e562be556e0fdfdaeb52df3b188